### ISSUE TITLE
Pinned Tabs?

### ISSUE THREAD

**Issue by burnoutberni** [PalmerAL/min#69]
IMHO [pinning tabs so that they are constantly displayed in the tab bar](https://support.mozilla.org/en-US/kb/pinned-tabs-keep-favorite-websites-open) is an essential feature.

**Comment by idle-zealot** [comment]
I agree that this would be nice, but as this browser doesn't display favicons, what should be shown in a pinned tab to indicate what page is loaded?

**Comment by burnoutberni** [comment]
the favicon? a favicon-colored box/circle with the first letter of the tab title? just a favicon-colored box/circle that displays changes to the tab title by blinking or changing color or something?

**Comment by whyhellobradley** [comment]
Safari doesn't have favicons until you pin a tab. Definitely, think this should be added.

**Comment by iambudi** [comment]
Any plan to implement this feature request near soon?
